src/modules/accounting/accounting.py

```diff
@@ -25,7 +29,50 @@ def execute_module(self, blockstamp: BlockStamp):
         """
         report_blockstamp = self.get_blockstamp_for_report(blockstamp)
         if report_blockstamp:
-            self.build_report(report_blockstamp)
+            self.process_report(report_blockstamp)
+
+        # Send third part data
+
+    @lru_cache(maxsize=1)
+    def build_report(self, blockstamp: BlockStamp) -> tuple:
+        return (
+            self.CONSENSUS_VERSION,
+            blockstamp.slot_number,
+            0,  # numValidators
+            0,  # clBalanceGwei
+            [],  # stakingModuleIdsWithNewlyExitedValidators
+            [],  # numExitedValidatorsByStakingModule
+            0,  # withdrawalVaultBalance
+            0,  # elRewardsVaultBalance
+            0,  # lastWithdrawalRequestIdToFinalize
+            0,  # finalizationShareRate
+            False,  # isBunkerMode
+            1,  # extraDataFormat
+            ZERO_HASH,  # extraDataHash
+            0,  # extraDataItemsCount
+        )
```

Comment:
suggest using some class with a method like `to_report()` that provides attributes values to the list
position values looks fragile (especially in boolean functions below)

```
@dataclass
class Report:
   version: int
   slot_number: int
...

   def to_report():
      return (self.version, self.slot_number, ...)
```

Reply:
Yep, I'll do it in accounting and ejector pr